src/dev_streamlit.py

```diff
@@ -0,0 +1,131 @@
+import time
+
+import numpy as np
+import pandas as pd
+import sys
+from pathlib import Path
+import streamlit as st
+import pydeck as pdk
+
+sys.path.append(str(Path(__file__).parent.parent))
+
+from src import api, cli
+
+# NOTE: This file is for testing purposes. Do not use it in production.
+
+# Page Configuration ###
+title = "cli-surf"
+st.set_page_config(
+    page_title=title,
+    page_icon="🌊",
+    layout="wide",
+)
+st.title(title)
+
+# Page Content ###
+
+# sidebar
+st.sidebar.markdown(
+    """
+# MENU
+- [weather](#weather)
+- [csv upload](#csv-upload)
+- [graph](#graph)
+- [data sheet](#data-sheet)
+""",
+    unsafe_allow_html=True,
+)
+
+# progress bar
+# st.write("Loading...")
+
+latest_iteration = st.empty()
+bar = st.progress(0)
+
+for i in range(100):
+    latest_iteration.text(f"Iteration {i + 1}")
+    bar.progress(i + 1)
+    time.sleep(0.01)
+
+# st.write("...and now we're done!")
+
+# Map
+# st.title("Surf Spot")
+
+st.caption("Enter a surf spot to see the map and forecast!")
+
+# User input location
+location = st.text_input("Surf Spot", placeholder="Enter surf spot!")
+
+# Checks if location has been entered. 
+# If True, gathers surf report and displays map
+if location:
+    location = "location=" + location
+    surf_report = cli.run(args=["placeholder",f"{location}"])
+    lat, long = surf_report["Lat"], surf_report["Long"]
+    map_data = pd.DataFrame(
+        np.random.randn(500, 2) / [50, 50] + [lat, long], columns=["lat", "lon"]
+    )
+    st.map(map_data)
```

Comment:
How about adding a `TODO:` comment to this so we don't forget?

Reply:
> With the current implementation, the map shows locations other than the ocean as well. It would be great if we could show only nearby surf spots based on the geocoding data.

Yep, I'm not happy with the map yet and I think thats a good idea. I'll add the `TODO`